code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -304,6 +305,7 @@
 			module_sprites["Eve"] = "eve"
 			module_sprites["Sleek"] = "sleekmedic"
 			module_sprites["#17"] = "servbot-medi"
+			module_sprites["Arachne"] = "aracne"
```

Comment:
How it was when I got the DMI and I didn't edit it, for some reason I don't
know

On Aug 20, 2016 11:40 AM, "ComicIronic" <notifications@github.com> wrote:

> In code/modules/mob/living/silicon/robot/robot.dm
> <https://github.com/d3athrow/vgstation13/pull/11499#discussion_r75582969>:
>
> > @@ -304,6 +305,7 @@
> >  			module_sprites["Eve"] = "eve"
> >  			module_sprites["Sleek"] = "sleekmedic"
> >  			module_sprites["#17"] = "servbot-medi"
> > +			module_sprites["Arachne"] = "aracne"
>
> Why is the icon_state spelt like that?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/d3athrow/vgstation13/pull/11499/files/54dac8c41098c78af8cef85ae96c041d6ad50e88#r75582969>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQMX7FtG3LZQQ2YKqlnr7vSWyxA6K6_Rks5qh0n8gaJpZM4JpCjF>
> .
>